src/cs/LionWeb.Integration.WebSocket.Tests/Node.js/RunNodeTests.cs

```diff
@@ -21,24 +21,25 @@ namespace LionWeb.Integration.WebSocket.Tests.Node.js;
 
 public class RunNodeTests
 {
-    private readonly ExternalProcessRunner _externalProcessRunner = new ();
+    private readonly ExternalProcessRunner _externalProcessRunner = new();
 
     [Test]
     public void RunNodeProgram()
     {
         _externalProcessRunner.StartProcess(
             "node",
             $"{Directory.GetCurrentDirectory()}/../../../Node.js",
-                // cwd is assumed to be: <repo root>/src/cs/LionWeb.Integration.WebSocket.Tests/bin/Debug/net8.0
-                // (hence 3x ../)
+            // cwd is assumed to be: <repo root>/src/cs/LionWeb.Integration.WebSocket.Tests/bin/Debug/net8.0
+            // (hence 3x ../)
             "node-program.js",
-            "started"
+            "started",
+            "Error"
```

Comment:
A second test that runs a Node.js program that errors out (shortly) after being started (and either staying running or stopping immediately) would test whether triggering on the `errorTrigger` works.

Reply:
Currently, this test is effectively the only unit test that asserts that `ExternalProcessRunner` works correctly. You added the `errorTrigger` argument to its `StartProcess` method, but there's no corresponding unit test that asserts that the extended functionality works correctly. An easy way would be to put more unit tests in this class, that asserts that the `errorTrigger` functionality works as expected. You'd probably need an additional Node.js program for that.